### PR TITLE
Add proper support for Mumble servers with no message length limit and fix not being able to embed from Mumble to other platforms

### PR DESCRIPTION
**Pull request by mvoolt**
I've also removed SetSelfMuted because it would send 2 event messages with that in.

### CODE REVIEW

File: bridge/mumble/mumble.go [hunk 1]
```diff
@@ -249,7 +249,7 @@ func (b *Bmumble) processMessage(msg *config.Message) {
 
 	// If there is a maximum message length, split and truncate the lines
 	var msgLines []string
-	if maxLength := b.serverConfig.MaximumMessageLength; maxLength != nil {
+	if maxLength := b.serverConfig.MaximumMessageLength; *maxLength != 0 {
```
Comment (42wim) [inline bridge/mumble/mumble.go:252]: Why do you remove the nil check? If the pointer is nil, it'll panic

Comment (mvoolt) [reply bridge/mumble/mumble.go]: I wasn't aware of that. This is my first time doing Golang.

Comment (42wim) [reply bridge/mumble/mumble.go]: No problem, why did you close it?

**Comment by mvoolt** [comment]
The merge branch commit felt inappropriate so I had to redo the pull request: #2008

**Pull request closed without merging**